.changelogs/2022-checkout.yml

```
@@ -0,0 +1,5 @@
+significance: patch
+type: fixed
+links: 
+- "#1997"
+entry: Fixed broken checkout on Twenty Twenty Two Theme when using password strength meter.
```

```suggestion
entry: Fixed broken checkout on Twenty Twenty-Two Theme when using password strength meter.
```

Source: https://github.com/WordPress/twentytwentytwo#twenty-twenty-two

---

booo